src/commons/repl/Repl.tsx

```diff
@@ -127,16 +125,6 @@ export const Output: React.FC<OutputProps> = (props: OutputProps) => {
   }
 };
 
-const renderResult = (value: any) => {
-  /** A class which is the output of the show() function */
-  const ShapeDrawn = (window as any).ShapeDrawn;
-  if (typeof ShapeDrawn !== 'undefined' && value instanceof ShapeDrawn) {
-    return <SideContentCanvasOutput canvas={value.$canvas} />;
```

Comment:
@chownces wanted to check if `ShapeDrawn` is still used? Found it being used in legacy `externLibs/.../webGL(rune|graphics).js`

> there are a couple of duplicated `renderResult` functions lying around as well, if `ShapeDrawn` is deprecated we can clean those up as well

Reply:
sry for the delayed response! i believe we can deprecate this now that runes has been migrated to the modules repo

Reply:
removed the use of the `ShapeDrawn` type, as for the deprecation of modules in `externalLibs`, i will batch it together in another PR